src/engine/enginemaster.h

```diff
@@ -133,13 +134,8 @@ class EngineMaster : public QObject, public AudioSource {
     }
 
     struct ChannelInfo {
-        ChannelInfo(int index)
```

Comment:
For these helper structs I prefer to keep their definition all in one place so you can easily look at them as a whole.